client/transaction-pool/graph/src/ready.rs

```diff
@@ -74,7 +83,7 @@ impl<Hash, Ex> PartialEq for TransactionRef<Hash, Ex> {
 }
 impl<Hash, Ex> Eq for TransactionRef<Hash, Ex> {}
 
-#[derive(Debug)]
+#[derive(Debug, parity_util_mem::MallocSizeOf)]
```

Comment:
I'd really just import it via `use parity_util_mem::MallocSizeOf` cause it's used quite frequently in this file.

Reply:
no longer the case